test/react-native/features/fixtures/ios-module/BugsnagModule.m

```diff
@@ -55,10 +55,21 @@ @implementation BugsnagModule
       NSLog(@"key: %@, value: %@ \n", key, [options objectForKey:key]);
   }
   BugsnagConfiguration *config = [[BugsnagConfiguration alloc] initWithApiKey:options[@"apiKey"]];
-  NSDictionary *endpointsIn = options[@"endpoints"];
-  NSString *notifyEndpoint = endpointsIn[@"notify"];
-  NSString *sessionsEndpoint = endpointsIn[@"sessions"];
+  NSString *notifyEndpoint;
+  NSString *sessionsEndpoint;
+  if (options[@"endpoints"] != nil && options[@"endpoints"][@"notify"] != nil && options[@"endpoints"][@"sessions"] != nil) {
+    NSDictionary *endpointsIn = options[@"endpoints"];
+    notifyEndpoint = endpointsIn[@"notify"];
+    sessionsEndpoint = endpointsIn[@"sessions"];
+  } else {
+    NSString *baseAddress = @"bs-local.com:9339";
```

Comment:
This will be replaced with the file loading logic in the below swift file once it can be compiled correctly